src/System.Security.Cryptography.Algorithms/tests/DefaultRSAProvider.cs

```diff
@@ -51,7 +51,7 @@ public bool Supports384PrivateKey
         public bool SupportsSha2Oaep
         {
             // Currently only RSACng does, which is the default provider on Windows.
-            get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows); }
+            get { return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && !(Create() is RSACryptoServiceProvider); }
```

Comment:
I don't know how often this property gets invoked, but that feels like an awful lot of GC fodder.